src/WebSdk/Publish/Targets/PublishProfiles/DefaultZipDeploy.pubxml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121. 
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <WebPublishMethod>ZipDeploy</WebPublishMethod>
+    <PublishProvider>AzureWebSite</PublishProvider>
```

Comment:
Can this one fall on the same case as MSDeploy, where the target can be different to AzureWebSite (on-prem)?

Reply:
ZipDeploy apis (in its current form) are only supported for Azure App Services and functions. It is not supported for other targets.